book/website/community-handbook/translation/translation-hello-crowdin.md

```diff
@@ -0,0 +1,165 @@
+(ch-translation-hello-crowdin)=
+
+## Crowdin 101
+
+Crowdin editor is your friend.
+You can use it to change translation language, proofread, add comments for contributors, contact the managers, vote translations, view suggestions for translation from translation memory or find Machine Translation from Google, Crowdin, DeepL, and others.
+
+### Hello Crowdin
+
+<<<<<<< HEAD
```

Comment:
fix conflicts until the end of file, remove all duplicated text and lines starting with `<<< HEAD` and `>>>>>>` cc @BatoolMM

Reply:
done